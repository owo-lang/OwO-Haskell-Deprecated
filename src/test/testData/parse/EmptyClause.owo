module EmptyClause where
 a : Type
 a = b
 module B where
  b : Type
  b = c
