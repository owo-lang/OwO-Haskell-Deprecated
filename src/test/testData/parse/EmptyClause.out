File type: owo
Top module name: .EmptyClause
 type signature a (26 2 2) (27 2 3)
  no pragmas
  named reference Type (30 2 6) (34 2 10)
 implementation of function a (36 3 2) (37 3 3)
  no pragmas
  clause for a (36 3 2) (37 3 3)
  patterns matching parsed as expression
   named reference a (36 3 2) (37 3 3)
  function body
   named reference b (40 3 6) (41 3 7)
  no with abstraction
  no where clause
 submodule .B
  type signature b (60 5 3) (61 5 4)
   no pragmas
   named reference Type (64 5 7) (68 5 11)
  implementation of function b (71 6 3) (72 6 4)
   no pragmas
   clause for b (71 6 3) (72 6 4)
   patterns matching parsed as expression
    named reference b (71 6 3) (72 6 4)
   function body
    named reference c (75 6 7) (76 6 8)
   no with abstraction
   no where clause
